filter.c

```diff
@@ -17,6 +21,25 @@ struct bpf_luaxdp_arg {
 	__u16 length;
 } __attribute__((packed));
 
+#ifdef DOME_MODE_ROUTER
+static inline int accept(void *data)
+{
+	return XDP_PASS;
+}
+#else /* DOME_MODE_BRIDGE */
+static inline int accept(void *data)
+{
+	unsigned char tmp[ETH_ALEN];
+	struct ethhdr *eth = data;
+
+	memcpy(tmp, eth->h_dest, ETH_ALEN);
+	memcpy(eth->h_dest, eth->h_source, ETH_ALEN);
+	memcpy(eth->h_source, tmp, ETH_ALEN);
+
+	return XDP_TX;
+}
+#endif
```

Comment:
```suggestion
static inline int accept(void *data)
{
#ifdef DOME_MODE_ROUTER
	return XDP_PASS;
#else /* DOME_MODE_BRIDGE */
	unsigned char tmp[ETH_ALEN];
	struct ethhdr *eth = data;

	memcpy(tmp, eth->h_dest, ETH_ALEN);
	memcpy(eth->h_dest, eth->h_source, ETH_ALEN);
	memcpy(eth->h_source, tmp, ETH_ALEN);

	return XDP_TX;
#endif
}
```

just to avoiding repeating a few lines.. feel free to ignore =)